source/common/http/conn_manager_config.h

```diff
@@ -212,6 +212,11 @@ class ConnectionManagerConfig {
    */
   virtual bool generateRequestId() PURE;
 
+  /**
+   * @return whether the x-request-id should be reset on edge entry inside mesh
+   */
+  virtual bool edgeAcceptRequestId() PURE;
```

Comment:
can you make this (and all the implementations for the derived classes) be const?  Up to you if you also want to fix generateRequestId() as well.

Reply:
I will change only my added method if that's ok with you